doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt

```diff
@@ -0,0 +1,12 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T13:13:20Z
+   2020-04-20T13:07:34Z
+   2020-04-20T10:55:42Z
```

Comment:
In the sake of consistency: `pbm list` displays history in reverse order - newest backups in the bottom. And backup history is always sorted in that order